modules/storage/sftp/sftp.go

```diff
@@ -41,23 +43,23 @@ type Params struct {
 	ConnectTimeout time.Duration
 }
 
-func Init(name string, params Params) (*SFTP, error) {
+func Init(name string, opts Opts, rl int64) (*SFTP, error) {
```

Comment:
Why the `name` and `rl` arguments are outside the `opts` structure?

Reply:
Because type conversion is used during initialisation. This point can be put in the refactoring task.